BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -1059,8 +1059,17 @@
 "CopyPrivateKey" = "Copy private key";
 "CopyFingerprint" = "Copy fingerprint";
 "SSHKeys" = "SSH keys";
+"ImportantNotice" = "Important notice";
+"BitwardenWillSendACodeToYourAccountEmail" = "Bitwarden will send a code to your account email to verify logins from new devices starting in February 2025.";
+"DoYouHaveReliableAccessToYourEmail" = "Do you have reliable access to your email, **%1$@**?";
+"YesICanReliablyAccessMyEmail" = "Yes, I can reliably access my email";
+"SetUpTwoStepLogin" = "Set up two-step login";
+"YouCanSetUpTwoStepLoginAsAnAlternative" = "You can set up two-step login as an alternative way to protect your account or change your email to one you can access.";
+"TurnOnTwoStepLogin" = "Turn on two-step login";
+"ChangeAccountEmail" = "Change account email";
 "NothingAvailableToAutofill" = "Nothing available to autofill";
 "FailedToGenerateVerificationCode" = "Failed to generate verification code";
 "FailedToAutofillItem" = "Failed to autofill item %1$@";
 "ExportingFailed" = "Exporting failed";
 "YouMayNeedToEnableDevicePasscodeOrBiometrics" = "You may need to enable device passcode or biometrics.";
+"TurnOnTwoStepLoginConfirmation" = "You can turn on two-step login on the bitwarden.com web vault. Do you want to visit the website now?";
```

Comment:
🤔 I was going off of the fact that the change email confirmation just says `bitwarden.com`. That's a good question, though.

Reply:
I guess this will change to not specify the host but to use "Bitwarden web vault" instead. Also I'd add `DescriptionLong` here as well to be consistent with other places.